src/app/admin/deleteUser/DeleteUserDialog.tsx

```diff
@@ -0,0 +1,81 @@
+import Button from "@mui/material/Button";
+import DeleteIcon from "@mui/icons-material/Delete";
+import React from "react";
+import { UserRow } from "@/app/admin/page";
+import styled from "styled-components";
+import Modal from "@/components/Modal/Modal";
+import { deleteUser } from "@/app/admin/adminActions";
+import OptionButtonsDiv from "@/app/admin/common/OptionButtonsDiv";
+import { SetAlertOptions } from "@/app/admin/common/SuccessFailureAlert";
+
+const DangerDialog = styled(Modal)`
+    & #deleteUserDialog {
```

Comment:
not a fan of selecting ID - should be reusable without 'knowing' to set the ID, and IDs can't be duplicated!